erizo/src/test/utils/Tools.h

```diff
@@ -0,0 +1,86 @@
+#ifndef ERIZO_SRC_TEST_UTILS_TOOLS_H_
+#define ERIZO_SRC_TEST_UTILS_TOOLS_H_
+
+#include <rtp/RtpHeaders.h>
+#include <MediaDefinitions.h>
+
+#include <string>
+#include <vector>
+
+namespace erizo {
+
+static constexpr uint16_t kVideoSsrc = 1;
+static constexpr uint16_t kAudioSsrc = 2;
+static constexpr uint16_t kArbitrarySeqNumber = 12;
+static constexpr uint16_t kFirstSequenceNumber = 0;
+static constexpr uint16_t kLastSequenceNumber = 65535;
+
+class Tools {
```

Comment:
What do you think about making this PacketTools and have more classes here if we need other things?